internal/custom_executable/head/head.go

```
@@ -0,0 +1,313 @@
+
+Mandatory arguments to long options are mandatory for short options too.
+  -c, --bytes=K     print the first K bytes of each file
+  -n, --lines=K     print the first K lines instead of the first 10
+  --help               display this help and exit
+  --version            output version information and exit`
+)
+
+type options struct {
+	lineCount   int
+	byteCount   int
+	quiet       bool
```

golangci-lint (1.64.8)

field `quiet` is unused

(unused)